Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml

```diff
@@ -1324,12 +1324,20 @@
   parent: VendingMachine
   name: vending machine
   abstract: true
+  placement:
+    mode: SnapgridCenter
+    snap:
+    - Wallmount
   components:
+  - type: Physics
+    canCollide: false
+  - type: Fixtures
   - type: Sprite
     drawdepth: WallMountedItems
     snapCardinals: false
+  - type: Rotatable
   - type: WallMount
-    arc: 180
+    arc: 175
   - type: Transform
     noRot: false
```

Comment:
Forgot to submit my review but these dummy physics components need removing.

Reply:
The reason it was added here is because they don't rotate properly w/out it due to inheriting physics features from base vending machines.

Reply:
Sorry, mistake...It's fixtures that was preventing it from rotating properly not the actual rotation but the access rotation. Meaning you cold rotate it N/E/W but could only access it from the South.

Physics makes it so you can attack the machine. W/out Physics, you can't hit it.